BridgeCareApp/VuejsApp/src/store-modules/inventory.module.ts

```diff
@@ -17,15 +17,22 @@ const mutations = {
 };
 
 const actions = {
-    async getInventory({commit}: any, payload: any) {
-        await new InventoryService().getInventory(payload.network)
+    async getInventory({ commit }: any) {
+        await new InventoryService().getInventory()
             .then((inventoryItems: InventoryItem[]) =>
                 commit('inventoryItemsMutator', inventoryItems)
             )
             .catch((error: any) => console.log(error));
     },
-    async getInventoryItemDetail({commit}: any, payload: any) {
-        await new InventoryService().getInventoryItemDetail(payload.inventoryItem)
+    async getInventoryItemDetailByBMSId({ commit }: any, payload: any) {
+        await new InventoryService().getInventoryItemDetailByBMSId(payload.bmsId)
+            .then((inventoryItemDetail: InventoryItemDetail) =>
+                commit('inventoryItemDetailMutator', inventoryItemDetail)
+            )
+            .catch((error: any) => console.log(error));
```

Comment:
We don't need a catch block on Promise code since we now have an axios interceptor that acts as a catch all for errors on the server.

Reply:
ok good to remove that?

Reply:
Yes, you can/should remove that anywhere you have it in your code for the inventory.

Reply:
Done.